cmd/exec/cmd.go

```diff
@@ -120,6 +121,13 @@ okteto exec api -- bash`,
 			}
 
 			if !okteto.IsOkteto() {
+				if manifest.Type == model.StackType {
+					return oktetoErrors.UserError{
+						E: fmt.Errorf("docker Compose format is only available using the Okteto Platform"),
+						Hint: `Follow this link to install the Okteto Platform in your Kubernetes cluster:
+    https://www.okteto.com/docs/get-started/install`,
+					}
+				}
```

Comment:
Wouldn't this check fit better within `ValidateForCLIOnly` function? I mean, as far as I know, docker-compose is not supported on CLI-only scenarios, independently of the command, so I think it makes sense to include it as part of that validation instead of spreading around the code different checks for the same case.

Also, in all the places where we call the `ValidateForCLIOnly`, we are adding this new check, and the docker-compose file is a manifest, so it fits well to be added there